requirements-dev.txt

```diff
@@ -1,3 +1,8 @@
-PyYAML
-pyright==1.1.257
-typing_extensions==4.2.0
\ No newline at end of file
+ipdb==0.13.11
```

Comment:
IMO we should get rid of this ipdb dependency (I've never used it, devs can set up IPython / ipdb themselves if they want it). Feel free to leave that for me in a follow-up PR though.

Reply:
I use it all the time, but I agree that it needn't be enforced by a project. Easy enough to install and use it myself.